app/scripts/abotypen/detail/abotypendetail.html

```diff
@@ -182,8 +182,8 @@ <h3 class="panel-title" translate>Zusatzdaten</h3>
                 <div class="input-group">
                   <div class="input-group-addon" ng-if="getModel() === 'abotypen'" >{{projekt.waehrung}}</div>
                   <div class="input-group-addon" ng-if="getModel() === 'zusatzAbotypen'" >{{projekt.waehrung}}</div>
-                  <input type="number" ng-if="getModel() === 'abotypen'" placeholder="{{'Preis'|translate}}" ng-pattern="/^[0-9]+([\.,][0-9]{1,3})?$/" step="0.01" class="form-control" name="zielpreis" ng-model="abotyp.zielpreis"></input>
-                  <input type="number" ng-if="getModel() === 'zusatzAbotypen'" placeholder="{{'Preis'|translate}}" ng-pattern="/^[0-9]+([\.,][0-9]{1,3})?$/" step="0.01" class="form-control" name="zielpreis" ng-model="zusatzAbotyp.zielpreis"></input>
+                  <input type="number" ng-if="getModel() === 'abotypen'" placeholder="{{'Preis'|translate}}" ng-pattern="/^[0-9]+([\.,][0-9]{1,3})?$/" step="0.01" min="0" max="99999.99" class="form-control" name="zielpreis" ng-model="abotyp.zielpreis"></input>
+                  <input type="number" ng-if="getModel() === 'zusatzAbotypen'" placeholder="{{'Preis'|translate}}" ng-pattern="/^[0-9]+([\.,][0-9]{1,3})?$/" step="0.01" min="0" max="99999.90" class="form-control" name="zielpreis" ng-model="zusatzAbotyp.zielpreis"></input>
```

Comment:
why is there a difference if max `99999.99` vs `99999.90` ?

Reply:
you are right, good catch, this should be 99. I am changing it